### ISSUE TITLE
glTF "metal" texture not displaying the same as FBX

### ISSUE THREAD

**Issue by ctrlaltdavid** [vircadia/vircadia#909]
![metals](https://user-images.githubusercontent.com/7455448/101991581-99902b00-3d12-11eb-976f-d3be904c15b6.png)

Note: The gltTF exporter for Blender also has a metallic multiplier.

**Comment by AleziaKurdis** [comment]
Here are the files:
http://metaverse.bashora.com/objects/pictureFrames/StainlessFrame.fbx
http://metaverse.bashora.com/objects/pictureFrames/StainlessFrame.glb

**Comment by SilverfishVR** [comment]
I have done some testing on this and it does indeed seem like the serializer is somewhat borked:

* Unpacking Alezias original glb I find `"metallicFactor": 0.99` and `"roughnessFactor": 0.13` not sure why roughness is lower, it could be color space issue in in that (older) gltf exporter, regardless, it validates and looks fine using babylon.js. In Vircadia it has correct roughness, but 0 metallic 😞 

* If I plug in solid color textures corresponding to the levels I want into both metal and roughness, it works and looks as expected. the glb contains a single magenta colored texture, red channel is roughness and blue is metallic 👍

* If I only plug in a roughness texture it appears to work but in fact it does not, the roughness value is not used and the metallic value is still derived from the ORM texture's blue channel, which is defaulted to pure white I think. it is the same if I only plug in metallic texture, value is ignored and corresponding channel of the ORM texture is used.

* I don't think the occlusionMap suffers from this issue, even though it is part of the ORM setup, maybe because there is no such thing as a occlusion value, it is always a texture

TL:DR the blender GLTF export is solid and all files I have tested validates and look right in babylon.js+tree.js, in Vircadia.. not so much 😄

test models used: [AleziaFrameTest](https://silverfish-freestuff.s3.eu-north-1.amazonaws.com/index.html?prefix=TestStuff/AleziaFrameTest/)

I used gltf seperate because it is easier to see what is going on but as far as I can see, it behaves the as glb.

tip: I use the GLTF tools plugin for VScode, it is very handy, it has 4 different render engines, validates with babylon.js and can even unpack glb's
